shakedown/dcos/master.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for working with master"""
+
+from shakedown import *
+
+
+def partition_master(incoming=True, outgoing=True):
+    """ Partition Master's port alone. To keep DC/OS cluster running.
+    """
+
+    print('Partitioning master. Incoming:{} | Outgoing:{}'
+        .format(incoming, outgoing))
+
+    if incoming and outgoing:
+        copy_file_to_master("{}/partition_master_cmd"
+            .format(shakedown_dcos_dir()))
+        run_command_on_master("sh partition_master_cmd")
```

Comment:
> is there a reason we chose to introduce bash scripts and copy them to the agents

No, and I think the `run_command_on_master` approach is cleaner.  I'll work on a patch for the code you referenced.